maps/manta.dmm

```diff
@@ -2442,11 +2442,11 @@
 /turf/space/fluid/manta,
 /area/abandonedmedicalship/robot_trader)
 "agq" = (
-/obj/machinery/door/airlock/pyro/security{
-	dir = 4;
-	lockdownbyai = 1;
```

Comment:
no more lockdownbyai?

Reply:
Reinstated, and applied to the main door too.